tp1.py

```diff
@@ -28,6 +28,7 @@ def test(model, data, target):
 
 def train(model, data, target, nEpoch, learningRate): #la training loop (iterate) 
     optim = torch.optim.Adam(model.parameters(), lr=learningRate) #ADAM variante de SGD
+    #optim=torch.optim.SGD(model.parameters(), lr=learningRate)
```

Comment:
C’est pas propre de laisser des codes en commentaire dans un projet. Vu que tu as fait les expériences, essaie de faire alors en sorte de passer l’optimiseur en paramètre en faisant une boucle for sur le nombre d’expériences et à l’interieur une boucle for pour le nombre d’optimiseur et à la fin tu les plot cote à cote. Tout ceci devra alors etre fait dans toytest()